types/mapbox__mapbox-sdk/index.d.ts

```diff
@@ -1,1752 +1,1753 @@
-        minLongitude: number;
-        minLatitude: number;
-        maxLongitude: number;
-        maxLatitude: number;
-    }
-
-    type GeocodeMode = 'mapbox.places' | 'mapbox.places-permanent';
-
-    type GeocodeQueryType =
-        | 'country'
-        | 'region'
-        | 'postcode'
-        | 'district'
-        | 'place'
-        | 'locality'
-        | 'neighborhood'
-        | 'address'
-        | 'poi'
-        | 'poi.landmark';
-
-    interface GeocodeRequest {
-        /**
-         * A location. This will be a place name for forward geocoding or a coordinate pair (longitude, latitude) for reverse geocoding.
-         */
-        query: string | LngLatLike;
-        /**
-         * Either  mapbox.places for ephemeral geocoding, or  mapbox.places-permanent for storing results and batch geocoding.
-         */
-        mode: GeocodeMode;
-        /**
-         * Limit results to one or more countries. Options are ISO 3166 alpha 2 country codes
-         */
-        countries?: string[];
-        /**
-         * Bias local results based on a provided location. Options are  longitude,latitude coordinates.
-         */
-        proximity?: number[];
-        /**
-         * Filter results by one or more feature types
-         */
-        types?: GeocodeQueryType[];
-        /**
-         * Forward geocoding only. Return autocomplete results or not. Options are  true or  false and the default is  true .
-         */
-        autocomplete?: boolean;
-        /**
-         * Forward geocoding only. Limit results to a bounding box. Options are in the format  minX,minY,maxX,maxY .
-         */
-        bbox?: BoundingBox;
-        /**
-         * Limit the number of results returned. The default is  5 for forward geocoding and  1 for reverse geocoding.
-         */
-        limit?: number;
-        /**
-         * Specify the language to use for response text and, for forward geocoding, query result weighting.
-         * Options are IETF language tags comprised of a mandatory ISO 639-1 language code and optionally one or more
-         * IETF subtags for country or script.
-         */
-        language?: string[];
-    }
+  import { LngLatLike } from 'mapbox-gl';
+  import { MapiRequest } from '@mapbox/mapbox-sdk/lib/classes/mapi-request';
+  import { MapiResponse } from '@mapbox/mapbox-sdk/lib/classes/mapi-response';
+  import MapiClient, { SdkConfig } from '@mapbox/mapbox-sdk/lib/classes/mapi-client';
+
+  /*********************************************************************************************************************
+   * Geocoder Types
+   *********************************************************************************************************************/
+
+  export default function Geocoding(config: SdkConfig | MapiClient): GeocodeService;
+
+  interface GeocodeService {
+    forwardGeocode(request: GeocodeRequest): MapiRequest;
+    reverseGeocode(request: GeocodeRequest): MapiRequest;
+  }
+
+  type BoundingBox = [
```

Comment:
All the blank space changes aside, this is the type change in question for this PR.

Reply:
👍